Common/Securities/IMarginCallModel.cs

```diff
@@ -46,6 +57,12 @@ public static class MarginCallModel
 
         private sealed class NullMarginCallModel : IMarginCallModel
         {
+            public SubmitOrderRequest GenerateMarginCallOrder(Security security, decimal netLiquidationValue, decimal totalMargin,
```

Comment:
I agree this work is for another PR, and that seems like the next reasonable improvement here. We'll want to do that before anyone starts writing their own margin models (probably rare and advanced users only, so they could probably handle the breaking change easily)